.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -104,11 +104,11 @@ body:
     id: workflow
     attributes:
       label: Workflow
-      description: How your application is managed? Not sure? Read [this part](https://docs.expo.dev/introduction/managed-vs-bare/) of Expo documentation
+      description: How your application is managed?
```

When I read this again it sounded weird 😅 

```suggestion
      description: How is your application managed?
```